pytorch_blade/src/compiler/mlir/converters/BUILD

```diff
@@ -43,10 +42,11 @@ cc_test(
     ],
     linkstatic = True,
     deps = [
-        "//src:libtorch_blade.so",
-        "@zlib",
+        ":torch_blade_mhlo_converter",
```

Comment:
Why change to static link?

Reply:
As discussed offline, this test should depend on //src:libtorch_blade.so to test agains the finally output shared library. Reverted.